node/parallel/src/chain_spec/heiko.rs

```diff
@@ -185,9 +185,9 @@ fn testnet_genesis(
             borrow_index: Rate::one(),                             // 1
             exchange_rate: Rate::saturating_from_rational(2, 100), // 0.02
             base_rate: Rate::saturating_from_rational(2, 100),     // 2%
-            kink_rate: Rate::saturating_from_rational(10, 100),    // 10%
+            jump_rate: Rate::saturating_from_rational(10, 100),    // 10%
```

Comment:
These model field can be grouped use a struct or tuple, can be reactored later.

Reply:
Sure